forc-plugins/forc-debug/src/cli/commands.rs

```diff
@@ -102,10 +94,12 @@ impl Commands {
         self.registers.name == cmd || self.registers.aliases.contains(&cmd)
     }
 
+    #[allow(unused)]
```

Comment:
can we just allow unused functions for this file at the top rather than on each individual fn?